co-log-benchmark/Main.hs

```diff
@@ -106,3 +109,13 @@ timeProcess n = do
       runProcess_ cfg
   t' <- getCurrentTime
   pure $ t' `diffUTCTime` t
+
+-- |  Function that takes list of pairs - benchmark name
+-- and result and generates markdown table.
+genTable pairs = intercalate "\n" rows
```

Comment:
It would be really nice for this function to have a type signature :slightly_smiling_face: